src/main/java/com/thed/zephyr/jenkins/reporter/ZeeReporter.java

```diff
@@ -187,7 +173,7 @@ public boolean perform(final Run build, final TaskListener listener) throws IOEx
 
             List<String> xmlFiles = new ArrayList<>();
 
-            if(Objects.equals(parserIndex, eggplantParserIndex)) {
+            if(Objects.equals(zephyrConfigModel.getParserTemplateId(), eggplantParserIndex)) {
```

Comment:
How these templates details are inserted into the Database?
eggplantParserTemplateId is 31 is it? (when we insert into db)

can you rename to eggplantParserTemplateId instead of index

Reply:
It is 3 long actually. Dependency of this variable will be removed in upcoming changes.